test/feature/features/move.feature

```diff
@@ -172,4 +159,26 @@ Feature: Move test
     And SQL result should not match regexp
     """
     002
-    """
\ No newline at end of file
+    """
+
+  Scenario: Move to non-existent shard fails
+    When I run SQL on host "coordinator"
+    """
+    MOVE KEY RANGE krid1 TO sh3
```

Comment:
Well, there is a high probability that sh3 will be present in future tests 😄
Let's name it something like `non-existent shard`